WINDOWS.md

```diff
@@ -0,0 +1,70 @@
+# Running Windows VM's
+* To run a VM as an agent, you'll need both SSH and Java installed on the Windows Image. We recommend using chocolatey to install OpenSSH and Java.
+
+## Creating a Windows Image
+* We will be using Packer to help us build a Windows 2016 image from GCE with OpenSSH and Java pre-installed. We will use the GCP cloud console, and you will need the following 2 files available to you on your google cloud console.
```

Comment:
Consider removing the uses of "we will" and "you will" for conciseness. More direct language is appropriate for this kind of documentation, eg: "Packer is used to build a Window 2016 image...", "the following 2 files are available on the google cloud console", etc.

Reply:
done